app/javascript/components/settings-details-tab/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import MiqDataTable from '../miq-data-table';
+import RegionForm from '../region-form';
+
+const settingsDetailsTab = ({
+  region, scanItemsCount, zonesCount, miqSchedulesCount,
+}) => {
+  const [rows, setRows] = useState([
+    {
+      id: '0',
+      region: {
+        text: `${region.description} [${region.region}]`,
+        icon: 'carbon--Db2Database',
+      },
+      treeBox: 'settings_tree',
+    },
+    {
+      id: '1',
+      region: {
+        text: `Analysis Profiles (${scanItemsCount})`,
```

Comment:
@liu-samuel I missed this during my last review but these strings are missing translations. Can you please mark these strings for translation.

Example:
``Analysis Profiles (${scanItemsCount})``

would become
``${__('Analysis Profiles')} (${scanItemsCount})``